Profiles/Core/profile_core.c

```diff
@@ -0,0 +1,130 @@
+#include "profile_core.h"
+
+#define MAX_PROFILE_NUMBER 10
```

Comment:
You should make it a real Luos configuration as other setup that can be on node_config.h